src/D2L.Bmx/Program.cs

```diff
@@ -102,6 +102,9 @@
 		result.ErrorMessage = $"Unsupported value for --output. Must be one of:\n{string.Join( '\n', PrintFormat.All )}";
 	}
 } );
+var cacheOption = new Option<int?>(
```

Comment:
Why int?

Reply:
We want to specify how much time is still left in the token. We don't want to give a token that expires the next second.

Reply:
Okay I see where it's used. From the name I would've thought it would be boolean and not expect to have to give a number here